three.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
```

Comment:
Your code gave 3 errors and a warning. After looking at your code, it's clear you must not be understanding functions. See Ms. Hsu or me as soon as possible; functions are very important to understand for the rest of this class and your computer science education.